cmd/neofs-node/notificator.go

```diff
@@ -127,8 +125,10 @@ func initNotifications(c *cfg) {
 			nats.WithRootCA(nodeconfig.Notification(c.appCfg).CAPath()),
 			nats.WithLogger(c.log),
 		)
-		if err != nil {
-			panic("could not created object notificator: " + err.Error())
+
+		endpoint := nodeconfig.Notification(c.appCfg).Endpoint()
+		if err := natsSvc.Connect(c.ctx, endpoint); err != nil {
```

Comment:
I thought that `Connect()` was meant to run somewhere else, e.g. in `bootUp()`. Correct me if I am wrong @carpawell